lib/tinybucket/resource/base.rb

```diff
@@ -17,7 +17,7 @@ def method_missing(method, *args)
       end
 
       def respond_to_missing?(symbol, include_all)
-        enum.respond_to_missing?(symbol, include_all) || super
```

Comment:
fixed two problems.

1. `enum`  is not defined in this scope. 😢 
2. I think, this `respond_to_missing?` should not depend on `super` because I want to forward the methods to the enumerator, not super.